src/flow/index.ts

```diff
@@ -0,0 +1,256 @@
+  sixth: (val: Sixth) => Seventh,
+  seventh: (val: Seventh) => Output
+): (val: First) => Output
+
+/**
+ * Lets you perform a flow of function operations sequentially
+ *
+ * @param functions - The function operations that you want to perform on a given value
+ * @returns A function that accepts an initial value to perform the above functions on
+ *
+ * @public
+ */
+export default function flow<
+  First extends unknown,
+  Second,
+  Third,
+  Fourth,
+  Fifth,
+  Sixth,
+  Seventh,
+  Eighth,
+  Output
+>(
+  first: (val: First) => Second,
+  second: (val: Second) => Third,
+  third: (val: Third) => Fourth,
+  fourth: (val: Fourth) => Fifth,
+  fifth: (val: Fifth) => Sixth,
+  sixth: (val: Sixth) => Seventh,
+  seventh: (val: Seventh) => Eighth,
+  eighth: (val: Eighth) => Output
+): (val: First) => Output
+
+/**
+ * Lets you perform a flow of function operations sequentially
+ *
+ * @param functions - The function operations that you want to perform on a given value
+ * @returns A function that accepts an initial value to perform the above functions on
+ *
+ * @public
+ */
+export default function flow<
+  First extends unknown,
+  Second,
+  Third,
+  Fourth,
+  Fifth,
+  Sixth,
+  Seventh,
+  Eighth,
+  Ninth,
+  Output
+>(
+  first: (val: First) => Second,
+  second: (val: Second) => Third,
+  third: (val: Third) => Fourth,
+  fourth: (val: Fourth) => Fifth,
+  fifth: (val: Fifth) => Sixth,
+  sixth: (val: Sixth) => Seventh,
+  seventh: (val: Seventh) => Eighth,
+  eighth: (val: Eighth) => Ninth,
+  ninth: (val: Ninth) => Output
+): (val: First) => Output
+
+/**
+ * Lets you perform a flow of function operations sequentially
+ *
+ * @param functions - The function operations that you want to perform on a given value
+ * @returns A function that accepts an initial value to perform the above functions on
+ *
+ * @public
+ */
+export default function flow<
+  First extends unknown,
+  Second,
+  Third,
+  Fourth,
+  Fifth,
+  Sixth,
+  Seventh,
+  Eighth,
+  Ninth,
+  Tenth,
+  Output
+>(
+  first: (val: First) => Second,
+  second: (val: Second) => Third,
+  third: (val: Third) => Fourth,
+  fourth: (val: Fourth) => Fifth,
+  fifth: (val: Fifth) => Sixth,
+  sixth: (val: Sixth) => Seventh,
+  seventh: (val: Seventh) => Eighth,
+  eighth: (val: Eighth) => Ninth,
+  ninth: (val: Ninth) => Tenth,
+  tenth: (val: Tenth) => Output
+): (val: First) => Output
+
+/**
+ * Lets you perform a flow of function operations sequentially
+ *
+ * @param functions - The function operations that you want to perform on a given value
+ * @returns A function that accepts an initial value to perform the above functions on
+ *
+ * @public
+ */
+export default function flow(...fns: AnyFn[]): AnyFn
+
+/**
+ * Lets you perform a flow of function operations sequentially
+ *
+ * @param functions - The function operations that you want to perform on a given value
+ * @returns A function that accepts an initial value to perform the above functions on
+ *
+ * @public
```

Comment:
```suggestion
 * @category Util
 * @public
```

Reply:
Added the test and made the above changes :)